apps/staking/components/StatusBar.tsx

```diff
@@ -222,49 +241,79 @@ export default function StatusBarProvider({ children }: { children?: ReactNode }
   const [networkStatusVisible, setNetworkStatusVisible] = useState<boolean>(false);
   const [networkInfo, setNetworkInfo] = useState<NetworkInfo | undefined>(undefined);
   const [networkInfoIsFetching, setNetworkInfoIsFetching] = useState<boolean>(false);
+  const [networkInfoIsLoading, setNetworkInfoIsLoading] = useState<boolean>(false);
   const [refetch, setRefetch] = useState<() => void>(() => {});
 
+  const { data: l2BlockNumber, refetch: refetchL2BlockNumber } = useBlockNumber();
+
+  const _refetch = () => {
+    refetch();
+    refetchL2BlockNumber();
```

Comment:
My linter tells me that `refetchL2BlockNumber` returns a promise. Does it not need to be awaited?

Reply:
Nah this doesn't need to be awaited. It triggers the refetch